test/stablehlo/test_exports.py

```diff
@@ -23,7 +23,7 @@ def __init__(self):
     super().__init__()
 
   def forward(self, a):
-    return a * torch.tensor(3)
+    return a / torch.tensor(3)
```

Comment:
it looks like a large change, and why do we change logic here from * to / ?

Reply:
This is a test case. So the error is that I got the order of constant vs. arg wrong. But because a* b = b*a so my old test case couldnt catch the error. but because a / b != b / a so errors of ordering shows up with this test case.